administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -103,9 +105,11 @@ COM_JOOMLAUPDATE_UPDATE_LOG_START="Update started by user %2$s (%1$s). Old versi
 COM_JOOMLAUPDATE_UPDATE_LOG_UNINSTALL_EXTENSIONS="Uninstalling extensions"
 COM_JOOMLAUPDATE_UPDATE_LOG_UPLOAD="Uploading update file"
 COM_JOOMLAUPDATE_UPDATE_LOG_URL="Downloading update file from %s."
+COM_JOOMLAUPDATE_MAIL_UPDATE_FAILED_DESC="Send to the site administrators if an Automated Joomla Update failed."
+COM_JOOMLAUPDATE_MAIL_UPDATE_FAILED_TITLE="Automated Joomla Update failed"
+COM_JOOMLAUPDATE_MAIL_UPDATE_SUCCESS_DESC="Send to the site administrators if an Automated Joomla Update succeeded."
+COM_JOOMLAUPDATE_MAIL_UPDATE_SUCCESS_TITLE="Automated Joomla Update succeeded"
```

```suggestion
COM_JOOMLAUPDATE_MAIL_UPDATE_SUCCESS_TITLE="Automated Joomla Update Succeeded"
```

capitalised for consistency